helita/sim/bifrost.py

```diff
@@ -357,8 +564,13 @@ def _get_simple_var(self, var, order='F', mode='r', *args, **kwargs):
             offset = (self.nx * self.ny *
                       (self.nzb + (self.nzb - self.nz) // 2) * idx * dsize)
             ss = (self.nx, self.ny, self.nz)
-        return np.memmap(filename, dtype=self.dtype, order=order, mode=mode,
-                         offset=offset, shape=ss)
+
+        if var in self.heliumvars:
+            return np.exp(np.memmap(filename, dtype=self.dtype, order=order,
```

Comment:
Using `np.exp` defeats the purpose of using `np.memmap`. Is there no other way of loading this?